packages/app/src/cli/services/context.ts

```diff
@@ -65,6 +66,8 @@ interface DevContextOutput {
   storeId: string
   updateURLs: boolean | undefined
   localApp: AppInterface
+  organization?: string
+  configFile?: string
```

Comment:
So I decided to have `ensureDevContext` return the `organization` and `configFile` to `logs` so we don't waste time and computation on computing it again.